kiali-operator/templates/kiali-cr.yaml

```diff
@@ -13,6 +13,9 @@ metadata:
   {{- include "kiali-operator.labels" . | nindent 4 }}
 annotations:
   ansible.sdk.operatorframework.io/verbosity: {{ .Values.debug.verbosity | quote }}
+  {{- if .Values.cr.annotations }}
+  {{- toYaml .Values.cr.annotations | nindent 2 }}
+  {{- end }}
```

Comment:
```suggestion
  annotations:
    ansible.sdk.operatorframework.io/verbosity: {{ .Values.debug.verbosity | quote }}
    {{- if .Values.cr.annotations }}
    {{- toYaml .Values.cr.annotations | nindent 4 }}
    {{- end }}
```

Reply:
Ha. I actually had a problem when I first added the the `cr.annotations` because I was indenting 4 and it was giving me an error. I realized it was only indenting 2 but totally missed that it was because it wasn't under `metadata`.